streams/lms_metadata.py

```diff
@@ -16,29 +16,34 @@ class LMSMetadataReader:
   # meta_ref is probably an unneccessary variable to pass as an arg since it's obscured from the user, but we can eventually make it an optional setting for the user
   def __init__(self, name: str, meta_ref: Optional[int] = 2, dump: Optional[bool] = False):
     self.player_name = name
-    self.IP = None
+    self.locale = None # locale replaces IP, it is a concatenation of both the IP and the port in {IP}:{port} format
```

Comment:
maybe just call this server? locale doesn't mean much 
A better comment might be just # {ip}:{port}